CODE_OF_CONDUCT.md

```diff
@@ -6,7 +6,7 @@ This code of conduct applies to all projects run by the Tendermint/COSMOS team a
 
 
 # Conduct
-## Contact: adrian@tendermint.com
+## Contact: ismail@tendermint.com
 
```

Comment:
@Liamsi let me know if this is okay? otherwise, I can put mine for the time being

Reply:
That's OK.